### ISSUE TITLE
Wrongly placed semicolon when calling function with herestring

### ISSUE THREAD

**Issue by netei** [eventualbuddha/decaffeinate#90]
Input : 

```
f=c("""line1
line2""")
```

Outputs: 

```
var f=c(`line1
line2`;)
```

Expected: 

```
var f=c(`line1
line2`);
```

**Comment by eventualbuddha** [comment]
[REPL](http://eventualbuddha.github.io/decaffeinate/repl/#?evaluate=true&code=f%3Dc(%22%22%22line1%0Aline2%22%22%22)) shows the same output. :+1:

**Comment by eventualbuddha** [comment]
Fixed in v1.41.1.
